woocommerce/woocommerce-android
# Switch to dedicated file on develop to upload strings for translation

As a part of our branch renaming projects, we'll need to update the path that GlotPress uses to fetch new strings that need translation.
Currently, the strings are picked from the main `strings.xml` file in the master branch. Upload from master assures that the changes applied during development don't affect the version in code freeze.
But this breaks our standard flow as it requires to merge the release branch into master multiple times during the release cycle.

This PR proposes a solution to this issue, following what we did on WPAndroid with https://github.com/wordpress-mobile/WordPress-Android/pull/12279. 
Instead of using a different branch, we can use a different file. The main `strings.xml` file is copied to a different location which gets updated only during the code freeze. After this is merged, we'll make GlotPress point to this shadow `strings.xml` in `develop` and we won't need to use `master` for this anymore. 

To test:
The easiest way to test this is to make the new `send_strings_for_translation` lane public:
1. Create a new branch out of this one.
2. Change `private_lane` to `lane` in `Fastfile`, line 401 so that the new lane can be called outside of the code freeze process.
3. Delete the new `fastlane/resources/values/strings.xml` file.
4. Run `bundle install`.
5. Run `bundle exec fastlane send_strings_for_translation`.
6. Verify that `fastlane/resources/values/strings.xml` has been created again. 

Update release notes:
